README.md

```diff
@@ -159,6 +161,9 @@ Usage: test-storybook [options]
 | `--ci`                            | Instead of the regular behavior of storing a new snapshot automatically, it will fail the test and require Jest to be run with `--updateSnapshot`. <br/>`test-storybook --ci` |
 | `--shard [shardIndex/shardCount]` | Splits your test suite across different machines to run in CI. <br/>`test-storybook --shard=1/3`                                                                              |
 | `--failOnConsole`                 | Makes tests fail on browser console errors<br/>`test-storybook --failOnConsole`                                                                                               |
+| `--includeTags`                   | Only test stories that match the specified tags, comma separated<br/>`test-storybook --includeTags="test-only"`                                                               |
```

Comment:
@yannbf one question for you, shouldn't these be tagged as experimental matching the filtering section that was added? Don't won't introduce confusion to anyone who is reading this, and start asking questions about it. Let me know what you think and we'll take it from there.

Reply:
how would you suggest this change be done?